source/howto/CreateNewImageFromUbuntuCloud.rst

```diff
@@ -0,0 +1,29 @@
+=============================
+Create New Image From Ubuntu Cloud
+=============================
+
+
+Documentations coming soon
+
+MT
+
+source/howto/CreateNewImageFromUbuntuCloud.rst
+
+.. code-block:: bash
+
+wget -o /var/lib/libvirt/boot/bionic-mini.iso \
```

Comment:
You need a tab indent on all the text you want in the code block to fix the waning:

```
.. code-block:: bash

  wget